src/gcp/run.ts

```diff
@@ -113,6 +126,9 @@ export interface IamPolicy {
   etag?: string;
 }
 
+/**
+ *
+ */
```

Comment:
nit* couple of unexpectedly empty docstring comments here and below.

Reply:
Will just document